autotest/gdrivers/wms.py

```diff
@@ -371,6 +371,15 @@ def test_wms_8():
     gdal.SetConfigOption("GDAL_DEFAULT_WMS_CACHE_PATH", None)
     assert cs == 0, 'cs != 0'
 
+    # Test with GDAL_DEFAULT_WMS_CACHE_TYPE
+    # Now, we should not read from the cache anymore
+    gdal.SetConfigOption("GDAL_DEFAULT_WMS_CACHE_TYPE", "none")
```

Comment:
``
with gdaltest.config_option("GDAL_DEFAULT_WMS_CACHE_TYPE", "none"):
     ...
`` is better now